option optimize_for = LITE_RUNTIME;

package OutOfBandBuffers;

message Instruction {
  optional uint64 seq_num = 1;
  optional uint64 ack_num = 2;
  optional uint64 oob_mode = 3;
  optional bytes payload_type = 4;
  optional bytes payload_data = 5;
}
